api/openapi-spec/swagger.json

```diff
@@ -10178,9 +10178,18 @@
      "migrations": {
       "$ref": "#/definitions/v1.MigrationConfiguration"
      },
+     "minCPUModel": {
+      "type": "string"
+     },
      "network": {
       "$ref": "#/definitions/v1.NetworkConfiguration"
      },
+     "obsoleteCPUModels": {
```

Comment:
Can this type be an `array` instead of `object`? It is a list of obsolete CPU models, so it should be ok.

Reply:
I chosen `map[string]bool` because then in code, it is much easier to check if model is obsolete or not. You don't have to iterate over slice for every model.

Reply:
If that is the only reason then please use an array. It will make the API easier to use, even if the implementation will need to use a loop. Iterating over a small number of elements is not a performance issue.